system_test/aest_hard_fork_SUITE.erl

```diff
@@ -734,6 +973,14 @@ load_module_on_node(NodeName, Module, String, Cfg) ->
           [Module, Binary]),
     ok = run_erl_cmd_on_node(NodeName, ErlCmd, 30000, Cfg).
 
+get_next_nonce(NodeName, PubKey, Cfg) ->
+    ErlCmd = lists:flatten(io_lib:format("aec_next_nonce:pick_for_account(~w).", [PubKey])),
```

Comment:
I understand the main portion of this PR is meant to be system test of contracts by reusing [positive test case of contract in integration test](https://github.com/aeternity/epoch/blob/700a1260/apps/aehttp/test/aehttp_integration_SUITE.erl#L638-L722). I see this test includes some improvements compared to the integration test - e.g. parametrization of inputs. Though it also keeps some of its shortcomings e.g. usage of internal commands in the node (rather than HTTP API a user would use). Would it be possible to improve on not using internal commands?

I am considering how this aest_hard_fork_SUITE could be reduced.